docs/chunking.ipynb

```diff
@@ -0,0 +1,903 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "1d53f86f-fa72-4161-a364-8c1f78dba6d6",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "\n",
+    "at_least_zero = lambda x: max(x, 0)\n",
+    "\n",
+    "chunk_long, chunk_lat = [10, 5]  # for int division\n",
+    "var = \"lat\"\n",
+    "\n",
+    "dss = []\n",
+    "dss_bitrounded = []\n",
+    "dss_kbits = []\n",
+    "\n",
+    "long_c = int(ds.lon.size / chunk_long)\n",
+    "lat_c = int(ds.lat.size / chunk_lat)\n",
+    "\n",
+    "for i in range(long_c):\n",
+    "    for j in range(lat_c):\n",
+    "        temp_ds = ds.isel(\n",
+    "            lon=slice(i * chunk_long, (i + 1) * chunk_long),\n",
+    "            lat=slice(j * chunk_lat, (j + 1) * chunk_lat),\n",
+    "        )\n",
+    "        dss.append(temp_ds)\n",
+    "        temp_info_pbit = xb.get_bitinformation(\n",
+    "            temp_ds, dim=var, implementation=\"python\"\n",
+    "        )\n",
+    "        temp_keepbits = xb.get_keepbits(temp_info_pbit, 0.99)\n",
+    "        # temp_keepbits = temp_keepbits.map(at_least_zero)\n",
+    "        dss_kbits.append(temp_keepbits)\n",
+    "        temp_ds_bitrounded = xb.xr_bitround(temp_ds, temp_keepbits)\n",
+    "        dss_bitrounded.append(temp_ds_bitrounded)\n",
+    "\n",
+    "        if i == 0 and j == 0:\n",
+    "            MERGED_ds_bitr = temp_ds_bitrounded\n",
+    "        else:\n",
+    "            MERGED_ds_bitr = xr.merge([MERGED_ds_bitr, temp_ds_bitrounded])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "7e1c4b35-4767-47e7-9ddf-357250b631b8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MERGED_ds_bitr.to_compressed_netcdf(\"3.air_chunked_bitr_compressed.nc\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3b60c66-252d-48a6-af93-a00c9ca8f0ba",
+   "metadata": {},
+   "source": [
+    "## ALL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "998581b5-6ad9-4f6f-9c61-d0bf1486ec7f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "7.5M\t0.air_original.nc\n",
+      "1.7M\t1.air_compressed_all.nc\n",
+      "1.3M\t2.air_bitrounded_compressed.nc\n",
+      "776K\t3.air_chunked_bitr_compressed.nc\n",
+      "1.1M\tair.zarr\n"
+     ]
+    }
+   ],
+   "source": [
+    "!du -hs *.nc *.zarr"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:bitinfo] *",
+   "language": "python",
+   "name": "conda-env-bitinfo-py"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.11.3"
+  }
```

Comment:
The `kernel spec` seem to cause the current CI error. Basically what is happening is that the kernel specifics with which you ran the notebook last are saved in this section so that the specific kernel can be loaded automatically next time. However, this specific kernel does not exist in our CI test. You could copy the metadata from the `quick_start.ipynb`:

```suggestion
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.4"
  },
```